### DISCUSSION TITLE
Reagarding the application update for github campus expert August 2022

### DISCUSSION THREAD

**Discussion by rudrakshkarpe** [community/community#33558] [Other Feature Feedback, Questions, & Ideas]
The application status date after essay submission was the 19th of September 2022, but there's still no update on it!

https://apply.githubcampus.expert/status

The URL above is broken for some reason, if anyone facing the same issues or any update with this please let me know in the discussion

![image](https://user-images.githubusercontent.com/78851635/191177769-d66426b4-e481-43c3-b820-0903598aba16.png)

**Comment by Nafkha** [comment]
Broken for me,
I think we are rejected, because there are some selected people
https://github.blog/2022-09-19-meet-the-github-campus-experts-selected-for-the-fall-2022-mlh-fellowship-cohort-powered-by-github/

**Reply by rudrakshkarpe** [reply]
This is the selection results for the MLH Fellowship fall Cohort, not the GitHub campus expert program

**Reply by rudrakshkarpe** [reply]
I hope for the same, time being let's be optimistic, hoping to get an official update form the GitHub

**Comment by Dharanash** [comment]
Yes, I also in the same situation! Can we get support from (https://support.github.com/).

**Reply by rudrakshkarpe** [reply]
Received a mail for rejection last night! I'll try next time with my best 💪

**Comment by melvincwng** [comment]
@rudrakshkarpe Hi Rudraksh & others, you should have received an email update from GitHub Education team with regards to your status of your application.

Can you kindly check if you've received any email updates from them?

Otherwise, please kindly contact [GitHub Education Support](https://support.github.com/contact/education) over here.

Hope this clarifies & answers your question 😊
